src/components/Settings/Notifications/NotificationsTelegram.tsx

```diff
@@ -64,6 +67,18 @@ const NotificationsTelegram = () => {
         /^-?\d+$/,
         intl.formatMessage(messages.validationChatIdRequired)
       ),
+    messageThreadId: Yup.string()
+      .when(['types'], {
```

Comment:
Didn't you forget a `'enabled'` here? (cf. line 59)

Reply:
I didn't as this field is optional. When `'enabled'` is added, the validation warning will show up after focusing the field once and not satisfying the regex pattern (i.e. leaving it empty). Right now the warning will only show up when the field is not empty and does not satisfy the pattern.